### ISSUE TITLE
[BUG] MazPhoneNumberInput typescript

### ISSUE THREAD

**Issue by seyfer** [LouisMazel/maz-ui#165]
**Describe the bug**
```
ERROR in /private/var/www/lumiform/webapp/node_modules/maz-ui/lib/index.d.ts(2,1):
2:1 An export assignment cannot be used in a module with other exported elements.
    1 | declare const _exports: any;
  > 2 | export = _exports;
      | ^
    3 | export var _MAZ_UI: any;
    4 |
Version: typescript 3.9.9
```

**To Reproduce**
```
import { PluginObject, VueConstructor } from 'vue';
// @ts-ignore
import { MazPhoneNumberInput } from 'maz-ui';

const MazUiPlugin: PluginObject<void> = {
  install(Vue: VueConstructor) {
    //@see https://louismazel.github.io/maz-ui/documentation/maz-phone-number-input
    Vue.use(MazPhoneNumberInput);
  },
};

export default MazUiPlugin;

...

Vue.use(MazUiPlugin);

...

<MazPhoneNumberInput />
```

**Expected behavior**
Just works

**Comment by seyfer** [comment]
@LouisMazel really critical one.

**Comment by seyfer** [comment]
there is a possible solution
https://stackoverflow.com/a/51238234/1074834

**Comment by seyfer** [comment]
I have disabled library check for now in my project because of this error
`"skipLibCheck": true`

but this is the wrong solution in the long run. I hope it will be fixed and then I can remove this config line.

**Comment by LouisMazel** [comment]
You can follow this PR https://github.com/LouisMazel/maz-ui/pull/166

**Comment by LouisMazel** [comment]
@seyfer 

Solved [v2.3.8](https://github.com/LouisMazel/maz-ui/releases/tag/v2.3.8)
